src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
:+1: from me! @ConnerMan -- can I ask in what context you're using IIIF? (email to azaroth42@gmail.com if you'd prefer) Thanks :)